src/v/model/record_batch_types.h

```diff
@@ -37,6 +37,8 @@ enum class record_batch_type : int8_t {
     node_management_cmd = 17, // controller node management
     data_policy_management_cmd = 18, // data-policy management
     archival_metadata = 19,          // archival metadata updates
+    cluster_config_delta_cmd = 20,   // cluster config incoming changes
```

Comment:
we may use single batch type for both config commands, they are differentiated by `command_type`

Reply:
Done